source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -124,6 +144,13 @@ void TextureBaker::bakeShaderInputs(ConstShaderRefPtr shaderRef, GenContext& con
         if (output)
         {
             FilePath filename = FilePath(outputFolder / generateTextureFilename(output, shaderRef->getName(), udim));
+
+            if (bindInput->getName() == "normal" && containsNormalMapNode(shaderRef))
```

Comment:
This is really close to the mark, though it would be even better if we could remove the dependency on the name "normal".  Would it be possible to scan each bindinput, regardless of name, and check the corresponding output for a connected <normalmap> node?

Reply:
Good point! I will try that.